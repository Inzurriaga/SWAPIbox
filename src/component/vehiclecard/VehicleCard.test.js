import React from "react"
import { shallow } from "enzyme"
import VehicleCard from "./Vehiclecard"

const mockdata = {
    name: "car",
    model: "ford",
    vehicle_class: "car",
    passengers: "4"
}


describe("vehicle_card", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(
            <VehicleCard vehicle={mockdata}/>
        )
    })
    it("should match snapshot", () => {
        expect(wrapper).toMatchSnapshot()
    })
})