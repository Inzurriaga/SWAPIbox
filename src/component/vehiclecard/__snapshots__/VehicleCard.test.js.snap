// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vehicle_card should match snapshot 1`] = `
<div
  className="vehicle-card"
>
  <h3>
    Name: 
    car
  </h3>
  <p>
    Model: 
    ford
  </p>
  <p>
    Class: 
    car
  </p>
  <p>
    Passengers: 
    4
  </p>
</div>
`;
